docs/api-reference/instances/attributeChange.md

```diff
@@ -0,0 +1,51 @@
+<nav class="fusiondoc-api-breadcrumbs">
```

Comment:
This file should be renamed to `attributechange.md` - I noticed you're probably copying the erroneous `doNothing.md`, which I'll adjust myself on the main branch shortly.